spec/std/deque_spec.cr

```diff
@@ -37,6 +37,22 @@ end
 
 private alias RecursiveDeque = Deque(RecursiveDeque)
 
+# Yields `Deque{}`, `Deque{0}`, `Deque{0, 1}`, `Deque{0, 1, 2}`, ...,
```

Comment:
Maybe this could help explain the behaviour a bit more?
```suggestion
# Yields multiple forms of `Deque{}`, `Deque{0}`, `Deque{0, 1}`, `Deque{0, 1, 2}`, ...,
```